addons/html_builder/static/src/snippets/add_snippet_dialog.js

```diff
@@ -81,14 +83,20 @@ export class AddSnippetDialog extends Component {
      */
     async insertStyle() {
         const loadCSSBundleFromEditor = (bundleName, loadOptions) => {
-            const cssLinkEl = this.props.editor.document.head
-                .querySelector(`link[type="text/css"][href*="/${bundleName}."]`);
+            const cssLinkEl = this.props.editor.document.head.querySelector(
+                `link[type="text/css"][href*="/${bundleName}."]`
+            );
             if (cssLinkEl) {
                 return loadCSS(cssLinkEl.getAttribute("href"), loadOptions);
             }
             return loadBundle(bundleName, loadOptions);
         };
-
+        for (const styleSheetGenerator of this.props.editor.getResource(
+            "extra_stylesheet_preview_dialog"
```

Comment:
Now that I reread the name: not a big fan of it but don't know what would a more appropriate name for it.